helix-rest/src/main/java/org/apache/helix/rest/server/service/InstanceServiceImpl.java

```diff
@@ -61,19 +61,32 @@
   private final HelixDataAccessorWrapper _dataAccessor;
   private final ConfigAccessor _configAccessor;
   private final CustomRestClient _customRestClient;
+  private boolean _skipZKRead;
```

Comment:
Should this field be `final`?

Reply:
Final requires constructor have redundant code. Let me try to refactor it.